ultra/ultra/config.yaml

```diff
@@ -26,8 +26,30 @@ tasks:
       train: "scenarios/taskX/train_taskX_bubbles*"
       test:  "scenarios/taskX/test_taskX_bubbles*"
     intersection-specific:
-      train: "scenarios/taskX/train_taskX_intersection-specific*"
-      test:  "scenarios/taskX/test_taskX_intersection-specific*"
+      train: "scenarios/taskX/train_intersection-specific*"
+      test:  "scenarios/taskX/test_intersection-specific*"
+    low-density:
+      train: "scenarios/taskX/train_low-density*"
+      train: "scenarios/taskX/test_low-density*"
+    mid-density:
+      train: "scenarios/taskX/train_mid-density*"
+      train: "scenarios/taskX/test_mid-density*"
+    high-density:
+      train: "scenarios/taskX/train_high-density*"
+      train: "scenarios/taskX/test_high-density*"
+  task0:
+    no-traffic:
+      train: "scenarios/task1/train_no-traffic*"
+      test:  "scenarios/task1/train_no-traffic*"
+    easy:
+      train: "scenarios/task1/train_easy*"
+      test:  "scenarios/task1/train_easy*"
+    medium:
+      train: "scenarios/task1/train_medium*"
+      test:  "scenarios/task1/train_medium*"
+    hard:
+      train: "scenarios/task1/train_hard*"
+      test:  "scenarios/task1/train_hard*"
```

Comment:
What is Task 0? I don't see it in our `scenarios/` directory. The paths seem to refer to Task 1 scenarios? Also, should these paths include the task in the scenario names?

Reply:
I do not know myself hahaha, I thought yesterday I had fixed that issue, but it seems that I was working on the same branch on two different repos, and I believe the changes were not correctly implemented. This also reflects on the comment below (task info in the scenario name). I have committed the new changes but I will push the changes once I have done a bit more training on the cross intersection